Tests/NetworkServiceTests/Extension/HTTPMethod+CaseIterable.swift

```diff
@@ -0,0 +1,10 @@
+import NetworkService
+
+extension HTTPMethod: CaseIterable {
+    public static var allCases: [HTTPMethod] {
```

Comment:
Should it be public var?

Reply:
The compile prompts that `yes`. 
<img width="989" alt="Screenshot 2023-06-28 at 09 01 28" src="https://github.com/ios-course/simple-network-service/assets/110284165/9ff5836c-070d-4993-a0fc-c2416ce26c3d">